Bottom: bd56db38e2bee27ae623faa67c56df829a4c8e59
Top:    f07ccac3291ccee0face25b581e0055095fa0ab1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:22:03 +0000

Rename hidGC "me_pointer" to "hid"

The pointer provides a link between a HID specific graphics context and HID
in question, name it accordingly.


---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index 3b2f01e..9ec7d32 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -1177,7 +1177,7 @@ gcode_make_gc (void)
   hidGC gc = (hidGC) calloc (1, sizeof (struct gcode_gc_struct));
   gcodeGC gcode_gc = (gcodeGC)gc;
 
-  gc->me_pointer = &gcode_hid;
+  gc->hid = &gcode_hid;
 
   gcode_gc->cap = Trace_Cap;
   gcode_gc->width = 1;
@@ -1265,7 +1265,7 @@ use_gc (hidGC gc)
 
   int need_brush = 0;
 
-  if (gc->me_pointer != &gcode_hid)
+  if (gc->hid != &gcode_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to gcode HID\n");
       abort ();
diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index b450568..8f2f661 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -940,7 +940,7 @@ gerber_make_gc (void)
   hidGC gc = (hidGC) calloc (1, sizeof (struct gerber_gc_struct));
   gerberGC gerber_gc = (gerberGC)gc;
 
-  gc->me_pointer = &gerber_hid;
+  gc->hid = &gerber_hid;
 
   gerber_gc->cap = Trace_Cap;
 
diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 9b679e5..6ebc746 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -125,7 +125,7 @@ ghid_make_gc (void)
   hidGC gc = (hidGC)g_new0 (struct gtk_gc_struct, 1);
   gtkGC gtk_gc = (gtkGC)gc;
 
-  gc->me_pointer = &ghid_hid;
+  gc->hid = &ghid_hid;
 
   gtk_gc->colorname = Settings.BackgroundColor;
 
@@ -495,7 +495,7 @@ use_gc (hidGC gc)
   render_priv *priv = gport->render_priv;
   GdkWindow *window = gtk_widget_get_window (gport->top_window);
 
-  if (gc->me_pointer != &ghid_hid)
+  if (gc->hid != &ghid_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to GTK HID\n");
       abort ();
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6f551c8..0c836eb 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -197,7 +197,7 @@ ghid_make_gc (void)
   hidGC gc = (hidGC) g_new0 (struct gtk_gc_struct, 1);
   gtkGC gtk_gc = (gtkGC)gc;
 
-  gc->me_pointer = &ghid_hid;
+  gc->hid = &ghid_hid;
 
   gtk_gc->colorname = Settings.BackgroundColor;
   gtk_gc->alpha_mult = 1.0;
@@ -525,7 +525,7 @@ ghid_invalidate_current_gc (void)
 static int
 use_gc (hidGC gc)
 {
-  if (gc->me_pointer != &ghid_hid)
+  if (gc->hid != &ghid_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to GTK HID\n");
       abort ();
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index bf35ffa..94e2142 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3059,7 +3059,7 @@ lesstif_make_gc (void)
 {
   hidGC gc = (hidGC)calloc (1, sizeof (struct lesstif_gc_struct));
 
-  gc->me_pointer = &lesstif_hid;
+  gc->hid = &lesstif_hid;
 
   return gc;
 }
@@ -3189,7 +3189,7 @@ set_gc (hidGC gc)
 {
   lesstifGC lesstif_gc = (lesstifGC)gc;
   int cap, join, width;
-  if (gc->me_pointer != &lesstif_hid)
+  if (gc->hid != &lesstif_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to lesstif HID\n");
       abort ();
diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index 90afd96..d11eb4f 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -765,7 +765,7 @@ nelma_make_gc(void)
 	hidGC gc = (hidGC) calloc (1, sizeof(struct hid_gc_struct));
 	nelmaGC nelma_gc = (nelmaGC)gc;
 
-	gc->me_pointer = &nelma_hid;
+	gc->hid = &nelma_hid;
 
 	nelma_gc->cap = Trace_Cap;
 	nelma_gc->width = 1;
@@ -848,7 +848,7 @@ use_gc(hidGC gc)
 	nelmaGC nelma_gc = (nelmaGC)gc;
 	int             need_brush = 0;
 
-	if (gc->me_pointer != &nelma_hid) {
+	if (gc->hid != &nelma_hid) {
 		fprintf(stderr, "Fatal: GC from another HID passed to nelma HID\n");
 		abort();
 	}
diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index 5f315e2..66bba9a 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -1352,7 +1352,7 @@ png_make_gc (void)
   hidGC gc = (hidGC) calloc (1, sizeof (struct png_gc_struct));
   pngGC png_gc = (pngGC)gc;
 
-  gc->me_pointer = &png_hid;
+  gc->hid = &png_hid;
 
   png_gc->cap = Trace_Cap;
   png_gc->width = 1;
@@ -1494,7 +1494,7 @@ use_gc (hidGC gc)
 
   int need_brush = 0;
 
-  if (gc->me_pointer != &png_hid)
+  if (gc->hid != &png_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to png HID\n");
       abort ();
diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index ea7dd55..773dfa9 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -424,7 +424,7 @@ eps_make_gc (void)
   hidGC gc = (hidGC) calloc (1, sizeof (struct eps_gc_struct));
   epsGC eps_gc = (epsGC)gc;
 
-  gc->me_pointer = &eps_hid;
+  gc->hid = &eps_hid;
 
   eps_gc->cap = Trace_Cap;
   eps_gc->width = 0;
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 96a00fb..f82b393 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1015,7 +1015,7 @@ ps_make_gc (void)
   hidGC gc = (hidGC) calloc (1, sizeof (struct ps_gc_struct));
   psGC ps_gc = (psGC)gc;
 
-  gc->me_pointer = &ps_hid;
+  gc->hid = &ps_hid;
   ps_gc->cap = Trace_Cap;
 
   return gc;
@@ -1101,7 +1101,7 @@ use_gc (hidGC gc)
       lastcap = lastcolor = -1;
       return;
     }
-  if (gc->me_pointer != &ps_hid)
+  if (gc->hid != &ps_hid)
     {
       fprintf (stderr, "Fatal: GC from another HID passed to ps HID\n");
       abort ();
diff --git a/src/hid_draw.h b/src/hid_draw.h
index 30e770c..8fadbf2 100644
--- a/src/hid_draw.h
+++ b/src/hid_draw.h
@@ -67,5 +67,5 @@ struct hid_draw_st
 
 /* Base hidGC elements visible to any module */
 struct hid_gc_struct {
-  HID *me_pointer;   /* Used by HIDs to validate the GCs passed belong to them */
+  HID *hid;   /* Used by HIDs to validate the GCs passed belong to them */
 };
